Magic/src/main/java/com/elmakers/mine/bukkit/action/builtin/SpellProgressAction.java

```diff
@@ -99,6 +104,15 @@ public SpellResult perform(CastContext context) {
                     if (requiredPathKey != null && currentPath == null) {
                         continue;
                     }
+
+                    //SkillAPI
+                    if(requiredSkillapiClass != null && !mage.hasSkillAPIClass(requiredSkillapiClass)){
+                        lore.add(ChatColor.RED + context.getMessage("skillapi_class_requirement").replace("$skillapiclass", requiredSkillapiClass));
```

Comment:
@NathanWolf The color should be in the language file right?

Reply:
Im not sure, if you look a few lines below it though for checking the required amount of casts, it does the same thing. I just decided to follow that

```
if (castCount == requiredCastCount) {
    lore.add(ChatColor.GREEN + context.getMessage("cast_requirement")
            .replace("$current", Long.toString(castCount))
            .replace("$required", Long.toString(requiredCastCount)));
} else {
    lore.add(ChatColor.RED + context.getMessage("cast_requirement")
            .replace("$current", Long.toString(castCount))
            .replace("$required", Long.toString(requiredCastCount)));
}
```